{"uuid": "3803e404-e7f8-4ffa-8364-04b8cc19b635", "children": ["4a7873c8-2de1-4c20-ab92-eec8232c6cc7"], "befores": [{"name": "set_up", "status": "passed", "start": 1568924050521, "stop": 1568924054397}], "afters": [{"name": "set_up::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 786, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\_python\\themis\\Tests/..\\Base\\base.py\", line 21, in set_up\n    self.driver.close()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\request.py\", line 68, in request\n    **urlopen_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\request.py\", line 89, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\poolmanager.py\", line 323, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 380, in _make_request\n    httplib_response = conn.getresponse()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 1331, in getresponse\n    response.begin()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 297, in begin\n    version, status, reason = self._read_status()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 258, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1568924077606, "stop": 1568924079783}], "start": 1568924050521, "stop": 1568924079783}